lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1156,6 +1156,77 @@ static void helperSetter(Constant *constT, std::vector<datatype> &vec) {
   }
 }
 
+template <typename T, typename datatype>
+static void helperSetter(const glow::Constant *constT,
+                         std::vector<datatype> &vec) {
+  auto constH = constT->getPayload().getHandle<T>();
+  for (size_t i = 0; i < constH.size(); ++i) {
+    vec.push_back(constH.at({i}));
+  }
+}
+
+Error ONNXModelLoader::loadRange(const ONNX_NAMESPACE::NodeProto &op,
+                                 ArgumentDictionaryTy &dict) {
+  const Constant *constT;
+  ASSIGN_VALUE_OR_RETURN_ERR(constT, getConstantByName(op.input(0)));
+
+  auto glowType = constT->getElementType();
+  Tensor T;
+  if (glowType == ElemKind::Int64ITy) {
```

Comment:
I think the code for int64 and int32 are essentially the same except for the datatype used -- can we create a shared templatized static helper to call, so the logic isn't duplicated? Or just use a macro to consolidate the logic in one place? Just seems silly to create almost exact copies.

Reply:
Added a template helper function.